src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE IF NOT EXISTS `book_shop`;
+    CREATE TABLE `books` (
+        `id` bigint NOT NULL AUTO_INCREMENT,
+        `title` varchar(225),
```

Comment:
Consider specifying a `NOT NULL` constraint for the `title` column if a book should always have a title. This will ensure data integrity by preventing null values.